features/alt-text-in-content.yml

```
@@ -0,0 +1,6 @@
+name: Alt text in content
```

I misunderstood this name at first. I parsed it as "alt text in [arbitrary] content" as if there was some new global `alt` attribute. Instead, what we have is alt text for the `content` property. I'm thinking there are two routes on the name here (since we don't have a lot of developer-originating language to go on):

- Take a nod from the spec

   ```suggestion
   name: Alt text for generated content
   ```

- Make the content property a disambiguation:

     ```suggestion
   name: Alt text (content property)
   ```

---

Good point- I went with "generated content" based on wording in a recent Mastodon thread on this.